python/py-praw/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-praw
+version             7.5.0
+supported_archs     noarch
+license             BSD
+
+python.versions     37 38 39 310
+
+maintainers         nomaintainer
+
+description         Python Reddit API Wrapper
+long_description    PRAW, an acronym for \"Python Reddit API Wrapper\", \
+                    is a Python package that allows for simple access to \
+                    Reddit's API. PRAW aims to be easy to use and internally \
+                    follows all of Reddit's API rules.
+
+homepage            https://github.com/praw-dev/praw/
+
+checksums           rmd160  37aef486547a08af9b0cd30581ed3a36ce22a1bc \
+                    sha256  f91bfc827ddd0718a6ec24f16f5cad0ea9279de82144e1afcbe47694d1e70cdb \
+                    size    22787277
+
+if {${name} ne ${subport}} {
+
+    depends_lib-append \
+                        port:py${python.version}-update_checker \
+                        port:py${python.version}-prawcore \
+                        port:py${python.version}-websocket-client
+
+    livecheck.type      none
```

Comment:
Also you can drop this line

Reply:
Hi @herbygillot,

I have made these changes.

Thanks,

-ranga